jenkins/pipelines/tracecompass-test.Jenkinsfile

```diff
@@ -181,13 +181,19 @@ pipeline {
             steps {
                 sshagent (['projects-storage.eclipse.org-bot-ssh']) { 
                     sh """
-                       ls -al ${RCP_DESTINATION}
-                       touch ${RCP_DESTINATION}test.txt
-                       ls -al ${RCP_DESTINATION}
-                       rm ${RCP_DESTINATION}test.txt
+                        SSHUSER="genie.tracecompass@projects-storage.eclipse.org"
+                        SSH="ssh ${SSHUSER}"
+                        SCP="scp"
+
+                        ${ssh} ls -al ${RCP_DESTINATION}
+                        ${ssh} touch ${RCP_DESTINATION}test.txt
+                        ${ssh} ls -al ${RCP_DESTINATION}
+                        ${ssh} rm ${RCP_DESTINATION}test.txt
+                        ${WORKSPACE_SCRIPTS}generate_download_page.sh ${RCP_DESTINATION} \"bogus/deploy/path\" \"${RCP_TITLE}\" > index.html
```

Comment:
isn't there a `/` missing after `$WORKSPACE_SCRIPTS`?

Reply:
I think there's already a trailing slash the way the variable is defined:
`WORKSPACE_SCRIPTS="${WORKSPACE}/.scripts/"`